src/common/Section/styled.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+    max-width: 1368px;
+    margin: 32px auto;
+    padding: 24px;
+`;
+
+export const Title = styled.h2`
+    color: ${({ theme }) => theme.colors.black};
+    font-size: 36px;
+    margin: 0;
```

Comment:
I would add margin-bottom instead of padding in body :)

Reply:
make sense), added margin-bottom instead of padding